src/GFExcelAdmin.php

```diff
@@ -559,19 +563,39 @@ private function download_count($form)
         return 0;
     }
 
+    /**
+     * Helper method to make the sort options field.
+     * @since $ver$
+     * @param array $form The form object.
+     */
     private function select_sort_field_options($form)
     {
-        $fields = array_merge([
+        $fields = array_reduce($form['fields'] ?? [], static function (array $fields, \GF_Field $field): array {
+            if (!$field->get_entry_inputs()) {
```

Comment:
Maybe add some comments to understand what is happening here. The function names aren't helpful enough.